source/connect.txt

```diff
@@ -0,0 +1,99 @@
+.. _java-rs-connect:
+
+==================
+Connect to MongoDB
+==================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :description: Learn how to use the Java Reactive Streams driver to connect to MongoDB.
+   :keywords: client, deploy, replica
+
+.. toctree::
+   :titlesonly:
+   :maxdepth: 1
+
+   /connect-to-mongo/create-a-mongo-client
+   /connect-to-mongo/choose-connection-target
+   /connect-to-mongo/connection-options
+
+Overview
+--------
+
+This page contains code examples that show how to connect your {+java-rs+} application
+to MongoDB with various settings.
+
+.. tip::
+
+  To learn more about the connection options on this page, see the link
+  provided in each section.
+
+To use a connection example from this page, copy the code example into the
+:ref:`sample application <java-rs-connect-sample>` or your own application.
+Be sure to replace all placeholders in the code examples, such as ``<hostname>``, with
```

Comment:
```suggestion
Be sure to replace all placeholders in the code examples, such as ``<Atlas connection string>``, with
```
There aren't any instances of <hostname> in the code examples on this page so changing the example here.